### PR TITLE
nautilus: update to 44.0.

### PR DESCRIPTION
**Pull request by someone13574**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-*
  - armv7l-*
  - armv6l-*
  - x86_64-musl
  - i686

### CODE REVIEW

File: srcpkgs/nautilus/template [hunk 1]
```diff
@@ -18,9 +18,9 @@ short_desc="GNOME file manager"
 maintainer="Michal Vasilek <michal@vasilek.cz>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
-changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-43/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/44.0/NEWS"
```
Comment (paper42) [inline srcpkgs/nautilus/template:21]: that's a tag, not branch, use master instead

File: srcpkgs/nautilus/template [hunk 2]
```diff
@@ -18,9 +18,9 @@ short_desc="GNOME file manager"
...
 distfiles="${GNOME_SITE}/nautilus/${version%%.*}/nautilus-${version}.tar.xz"
-checksum=a86a9aa7a455511b029ce49a1d8b4659c3c355b5d81d7360bb4d0de637afede0
+checksum=57b99ebb8e2b9c1512d381e53096188c087b33410d6c52d87a27bd60ee76ac7f
 # tests timeout:
 # 11/11 test-nautilus-search-engine-tracker            TIMEOUT        480.00s
 make_check=extended
```
Comment (paper42) [inline srcpkgs/nautilus/template:26]: this test takes 0.4s for me, can we remove this?

Comment (someone13574) [reply srcpkgs/nautilus/template]: CI fails without it, local build works fine

Comment (paper42) [reply srcpkgs/nautilus/template]: then we should use ci-skip instead of extended

**Pull request merged**